tools/SetupFlow/DevHome.SetupFlow/ViewModels/AddRepoViewModel.cs

```diff
@@ -350,6 +420,37 @@ private void MenuItemClick(string selectedItemName)
         });
     }
 
+    [RelayCommand]
+    public void SortRepos(TextBlock selectedItem)
+    {
+        IEnumerable<IRepository> repositories = new List<IRepository>();

```

Comment:
I believe this needs to stay this way because line `OrderBy` and `OrderByDescending` return an IOrderedEnumerable.  I can't make a `new` IEnumerable.

I could type `var repositories = new List<IRepository>() as IEnumerable<IRepository>`  I'll leave this as is.